lib/Button/Button.js

```diff
@@ -268,5 +279,16 @@ module.exports = kind(
 		if (!this.showing && this.hasClass('pressed')) {
 			this.undepress();
 		}
+	},
+
+	/**
+	* @private
+	*/
+	backgroundOpacityChanged: function (old) {
+		var opacity = this.get('backgroundOpacity');
```

Comment:
Unfortunately, that will fail on the call during `create()` because it's called without args. At a minimum, we should change it to `this.backgroundOpacity` to avoid the unnecessary `get()` call. I'm all for using `get()` from outside a kind to protect the interface but from within I'd avoid it for the perf benefit.

On the following line, i would add an existence check to again skip an unnecessary function call.

```
if (old) this.removeClass(old);
```